### ISSUE TITLE
Provide deeper descriptions on services

### ISSUE THREAD

**Issue by rgbkrk** [jrperritt/rack#156]
Our descriptions are redundant and cyclic. It would be nice to flesh out the description for people exploring the client. Take block storage as an example:

```
$ rack block-storage help
NAME:
   rack block-storage - Used for the Block Storage service
```

How's this:

```
$ rack block-storage help
NAME:
   rack block-storage - Block level storage, exposed as volumes to mount to host servers.
                        Work with volumes and their associated snapshots.
```

**Comment by kenperkins** [comment]
I very much like the idea of making the descriptions more friendly and verbose.

**Comment by rgbkrk** [comment]
```
$ rack block-storage snapshot list
Nothing to show. Maybe you'd like to set up some snapshots?
```

Ha, I liked this one.

**Comment by jrperritt** [comment]
Is this closed by #157 ?

**Comment by rgbkrk** [comment]
Yup!
